File: Linguagem C/exercicios listas/Soma_Vetor.c
#include <stdio.h>

int main()
{
    int N, i,cont;
    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    double media,soma,vet[N];

    soma = 0;
    media = 0;
    cont = 0;

    for (i = 1; i <= N; i++)
    {
        printf("Digite um numero: ");
        scanf("%lf",&vet[i]);
    }

    for (i = 1; i <= N; i++)
    {
        soma += vet[i];
        cont += 1;
    }

    media = soma/cont;

    printf("VALORES = ");

    for (i = 1; i <= N; i++)
    {
        printf("%.1lf  ",vet[i]);
    }

    printf("\nSOMA =  %.2lf\n",soma);
    printf("MEDIA = %.2lf\n",media);

    return 0;
}

File: Python/exercicios propostos/soma_vetor.py
num = int(input("Quantos numeros voce vai digitar?: "))

soma = 0
media = 0


vet = [0 for x in range (num)]

for i in range (0,num):
  vet[i] = float(input("Digite um numero: "))
  soma += vet[i]

  
media = soma/num

print("")
print("VALORES = ",end =' ')

for i in range (0,num):
  print(f"{vet[i]:.1f}", end = ' ')
  
print("")
print(f"SOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")